src/github.com/getlantern/flashlight/app/pac.go

```diff
@@ -197,16 +197,22 @@ func doPACOn(pacURL string) {
 	//
 	// By changing the URL here we are forcing the OS to check the URL whenever
 	// Lantern starts.
-	pacURLNoCache = fmt.Sprintf("?%d", time.Now().UnixNano())
+	noCache := fmt.Sprintf("?%d", time.Now().UnixNano())
+	pacURLNoCache.Store(noCache)
 
-	err := pac.On(pacURL + pacURLNoCache)
+	err := pac.On(pacURL + noCache)
 	if err != nil {
 		log.Errorf("Unable to set lantern as system proxy: %v", err)
 	}
 }
 
 func doPACOff(pacURL string) {
-	err := pac.Off(pacURL + pacURLNoCache)
+	var noCache string
+	_noCache := pacURLNoCache.Load()
+	if _noCache != nil {
+		noCache = _noCache.(string)
+	}
+	err := pac.Off(pacURL + noCache)
```

Comment:
This looks fine @oxtoacart. I know it's not a part of this PR, but it does seem like just storing the whole URL without a separate noCache state would make more sense (just store pacURL with the query string in the atomic value) just to eliminate an extra variable, but not a huge deal.